app.js

```diff
@@ -1,10 +1,15 @@
 const express = require('express')
 const { getTopics } = require('./controllers/topics.controller')
+const endpoints = require('./endpoints.json')
 
 const app = express()
 
 app.get('/api/topics', getTopics)
 
+app.get('/api', (request, response, next) => {
+    response.status(200).send(endpoints)
```

Comment:
Responses are usually sent from the controller so, for uniformity, it would be beneficial to send this from a function in the controller